WSSiOS/WSSiOS/Source/Presentation/Base/AlertButton/WSSAlertButtonView.swift

```diff
@@ -160,12 +174,16 @@ extension WSSAlertButtonView {
         
         if let actionTitle {
             actionButton.do {
-                $0.setTitle(actionTitle, for: .normal)
-                $0.setTitleColor(.wssWhite, for: .normal)
-                $0.titleLabel?.font = .Label1
                 $0.layer.backgroundColor = actionBackgroundColor
                 $0.layer.cornerRadius = 8
             }
+            
+            actionButtonTitleLabel.do {
+                $0.isUserInteractionEnabled = true
```

Comment:
W1; 
```suggestion
                $0.isUserInteractionEnabled = false
```

로 해줘야 버튼 액션 활성화! 요거만 빠루게 수정 부탁드려용

Reply:
나 왜저랬대 ㅋ

Reply:
나 왜저랬대